cluster/juju/OWNERS

```diff
@@ -4,13 +4,14 @@ approvers:
 - marcoceppi
 - mbruzek
 reviewers:
+- chuckbutler
```

Comment:
@nikhiljindal not for these directories. He helps us with documentation and I believe is a reviewer/approver on our docs directory.